test/over_react/component_declaration/builder_integration_tests/new_boilerplate/function_component_test.dart

```diff
@@ -381,7 +385,22 @@ UiFactory<TestProps> Test = uiFunction(
       ..addProp('data-prop-custom-key-and-namespace-prop',
           props.customKeyAndNamespaceProp))('rendered content');
   },
-  $TestConfig, // ignore: undefined_identifier
+  _$TestConfig, // ignore: undefined_identifier
+);
+
+UiFactory<TestProps> TestPublicConfig = uiFunction(
+      (props) {
+    return (Dom.div()
+      ..addTestId('testId')
+      ..addProp('data-prop-string-prop', props.stringProp)
+      ..addProp('data-prop-dynamic-prop', props.dynamicProp)
+      ..addProp('data-prop-untyped-prop', props.untypedProp)
+      ..addProp('data-prop-custom-key-prop', props.customKeyProp)
+      ..addProp('data-prop-custom-namespace-prop', props.customNamespaceProp)
+      ..addProp('data-prop-custom-key-and-namespace-prop',
+          props.customKeyAndNamespaceProp))('rendered content');
+  },
+  $TestConfig, // ignore: undefined_identifier, deprecated_member_use_from_same_package
```

Comment:
This should use its own config 
```suggestion
  $TestPublicConfig, // ignore: undefined_identifier, deprecated_member_use_from_same_package
```